src/store/index.ts

```diff
@@ -173,11 +194,21 @@ const useTaxesStore = defineStore({
           : 0;
 
       return (
-        grossIncome *
-          (this.firstYear ? 0.375 : this.secondYear ? 0.5625 : 0.75) +
+        (grossIncome - this.youthIrsDiscount) *
+        (this.firstYear ? 0.375 : this.secondYear ? 0.5625 : 0.75) +
```

Comment:
Are we sure we want to apply the first and second year discount on top of the youth IRS discount?

According to [this article](https://www.comparaja.pt/blog/irs-para-trabalhadores-independentes-coverflex#:~:text=nas%20categorias%20anteriores.-,No,-caso%20dos%20rendimentos) this "bonificação" is not related with youth IRS. So I'm not sure if they are cumulative or not. Do you know?

Reply:
or worse: mutually exclusive. I enabled IRS Jovem on my second year IRS but it wasn't transparent to me if it was cumulative or if, being mutually exclusive, they defaulted to the most worth option for the contributor. Without any source to back this up I think it would be best to assume mutually exclusivity, that way in the UI the user can simulate which option is more worth to him. If it turns out it is cumulative than even better and the user will have a (good) surprise when it comes to pay the IRS :)

Reply:
😨 that's terrible! But I agree with your reasoning. Still impressed that there's no clear documentation for this scenarios in oficial platforms!

Reply:
Exactly.. One can only imagine what the actual implementation looks like, since it's not publicly accessible... For all it's worth it might not be even implemented... Or worse, wrongfully implemented! There's no formal specification or proof that it is correct.. If there's no clear official documentation I can't understand what it must have been for the developers who had to implement it 😅

Reply:
Perfect, thanks. In that case, if you remove the extra log (comment above) it's good to go!

Reply:
I think it's very hard to know that. If one of you finds some documentation on this, let me know